src/main/java/com/intuit/graphql/orchestrator/GraphQLOrchestrator.java

```diff
@@ -80,12 +81,12 @@ private DataLoaderRegistry buildNewDataLoaderRegistry() {
   }
 
   public CompletableFuture<ExecutionResult> execute(ExecutionInput executionInput) {
-    return execute(executionInput, false);
+    return execute(executionInput, null, false);
```

Comment:
I suggest avoid null/false since it does not tell the meaning or intent.  Maybe something like:

```
execute(executionInput, DEFAULT_DEFER_OPTIONS, HAS_DEFER_FALSE). 
```